src/js/tracks/text-track.js

```diff
@@ -420,6 +421,16 @@ class TextTrack extends Track {
     this.cues.setCues_(this.cues_);
   }
 
+  /**
+   * Makes the text track serializable by removing circular dependencies.
```

Comment:
I think this description should make clear that it returns a new object rather than mutating the track in place